### PR TITLE
DOCS: Updates to Documentation to include recent improvements to extension

### PR DESCRIPTION
**Pull request by mmcky**
This PR updates documentation for `sphinxcontrib-jupyter`.

### CODE REVIEW

File: docs/Manual.rst [hunk 1]
```diff
@@ -0,0 +1,844 @@
...
+=======
+
+This project is supported by `QuantEcon <https://www.quantecon.org>`__
+
+Many thanks to the contributors of this project.
+
+* `@mmcky <https://github.com/mmcky>`__
+* `@myuuuuun <https://github.com/myuuuuun>`__ 
+* `@NickSifniotis <https://github.com/NickSifniotis>`__
+
```
Comment (mmcky) [inline docs/Manual.rst]: please add yourself to this when you get a chance

**Comment by mmcky** [comment]
hey @FelipeMaldonado I think I will split this out of a single file and into section files so that we have different pages (rather than one long page).

**Comment by mmcky** [comment]
thanks @FelipeMaldonado. I have reworked this a little to better fit in with `rtd` etc.

**Comment by FelipeMaldonado** [comment]
All good @mmcky, I was checking the current version and it looks good. Once you review the static-folder PR I'll #134  I can include the documentation for that as well.

**Pull request merged**